deploy/olm-catalog/ocs-operator/0.0.1/storagecluster.crd.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: storageclusters.ocs.openshift.io
+spec:
+  group: ocs.openshift.io
+  names:
+    kind: StorageCluster
+    listKind: StorageClusterList
+    plural: storageclusters
+    singular: storagecluster
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            instanceType:
+              type: string
+            manageNodes:
+              description: 'Important: Run "operator-sdk generate k8s" to regenerate
+                code after modifying this file'
+              type: boolean
+            storageDeviceSets:
+              items:
+                properties:
+                  config:
+                    type: object
+                  count:
+                    format: int64
+                    type: integer
+                  name:
+                    type: string
+                  placement:
+                    properties:
+                      nodeAffinity:
+                        type: object
+                      podAffinity:
+                        type: object
+                      podAntiAffinity:
+                        type: object
+                      tolerations:
+                        items:
+                          type: object
+                        type: array
+                    type: object
+                  resources:
+                    type: object
+                  volumeClaimTemplates:
+                    type: object
+                required:
+                - name
+                - count
+                - resources
+                - placement
+                - volumeClaimTemplates
+                type: object
+              type: array
+          required:
+          - storageDeviceSets
```

Comment:
If I understand this correctly, we don't want to do this. We want to be able to create a StorageCluster with no StorageDeviceSets.